tests/core/test_opds2_import.py

```diff
@@ -71,36 +78,52 @@ def _get_work_by_identifier(works, identifier):
 
 
 class TestOPDS2Importer(OPDS2Test):
+    MOBY_DICK_IDENTIFIER = "urn:isbn:978-3-16-148410-0"
+    HUCKLEBERRY_FINN_IDENTIFIER = "http://example.org/huckleberry-finn"
+
     def sample_opds(self, filename, file_type="r"):
         base_path = os.path.split(__file__)[0]
         resource_path = os.path.join(base_path, "files", "opds2")
         return open(os.path.join(resource_path, filename)).read()
 
+    def setup_method(self):
+        super(TestOPDS2Importer, self).setup_method()
```

Comment:
Minor: We can simplify to `super().setup_method()` here? But we can clean these up with `pyupgrade` linter.

Reply:
I can change this quick, no probkems

Reply:
This is actually everywhere, I'd rather leave it to the pyupgrade tool